schemas/create-aliquot-in-mlwh/1-create-aliquot-in-mlwh.avsc

```diff
@@ -0,0 +1,124 @@
+{
+    "type": "record",
+    "name": "CreateAliquotInMLWH",
+    "namespace": "uk.ac.sanger.psd",
+    "doc": "Create an Aliquot in MLWH coming from Traction.",
+    "fields": [
+        {
+            "name": "messageUuid",
+            "type": {
+                "type": "string",
+                "logicalType": "uuid"
+            },
+            "doc": "Unique message ID."
+        },
+        {
+            "name": "messageCreateDateUtc",
+            "type": {
+                "type": "long",
+                "logicalType": "timestamp-millis"
+            },
+            "doc": "Date (UTC) that the message was created."
+        },
+        {
+            "name": "limsId",
+            "type": "string",
+            "doc": "The LIMS system that the aliquot was created in"
+        },
+        {
+            "name": "limsUuid",
+            "type": {
+                "type": "string",
+                "logicalType": "uuid"
+            },
+            "doc": "The UUID of the aliquot in the LIMS system"
+        },
+        {
+            "name": "aliquotType",
+            "type": {
+                "type": "enum",
+                "name": "allowedAliquotType",
+                "symbols": [
+                    "primary",
+                    "derived"
+                ]
+            },
+            "doc": "The type of the aliquot"
+        },
+        {
+            "name": "sourceType",
+            "type": {
+                "type": "enum",
+                "name": "allowedSourceType",
+                "symbols": [
+                    "well",
+                    "sample",
+                    "library",
+                    "pool"
+                ]
+            },
+            "doc": "The type of the source of the aliquot"
+        },
+        {
+            "name": "sourceBarcode",
+            "type": "string",
+            "doc": "The barcode of the source of the aliquot"
+        },
+        {
+            "name": "sampleName",
+            "type": "string",
+            "doc": "The name of the sample that the aliquot was created from"
+        },
+        {
+            "name": "usedByType",
+            "type": {
+                "type": "enum",
+                "name": "allowedUsedByType",
+                "symbols": [
+                    "library",
+                    "pool",
+                    "run",
+                    "none"
+                ],
+                "default": "none"
```

Comment:
No worries! 

I think @seenanair did some tweaks to the schema based on a discussion. I cannot guarantee that I know 100% of what "none" stands for but I remember a quick discussion in the standup. Maybe @seenanair could give a hint?

Reply:
Aliquot creation is done in different contexts, out of which one of it is creating a 'primary' aliquot when a library is created. In this scenario, the aliquot is of type 'primary' and it will only have source and no used_by. 
Another context is when a library is used in a pool or run, we are creating a 'derived' aliquot. In this case, the aliquot is of type 'derived' and it will have source (library) and used_by (pool/run well).
So, it is not mandatory that used_by field is always filled up especially in first scenario.

I have made this change, after Stuart mentioned about an error where the  'none'  for 'usedByType' was not in the correct place to make it a default value.
`{
            "name": "usedByType",
            "type": {
                "type": "enum",
                "name": "allowedUsedByType",
                "symbols": [
                    "library",
                    "pool",
                    "run",
                    "none"
                ]
            },
            "default": "none",
            "doc": "The type of the entity that the aliquot is used by"
        },`

Reply:
So we just need to make sure the schema here matches what is on RedPanda, if we even want to keep maintaining a second copy of schemas in this repo